tests/integration/incremental_on_schema_change/models/incremental_ignore_target.sql

```diff
@@ -1,5 +1,5 @@
-{{ 
-    config(materialized='table') 
+{{
+    config(materialized='table')
```

Comment:
I guess the black formatter doesn't recognize the quotes in Jinja files.

Reply:
Yes, those files are out of scope of black. We have to manually handle those for now.